spec/in_paralell_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rspec'
 require_relative('../lib/in_parallel')
 include InParallel
-TMP_FILE = '/tmp/test_file.txt'
+TMP_FILE = 'tmp/test_file.txt'
```

Comment:
Oh oops I did not look carefully enough, I thought the link you had was to Ruby's tempfile http://ruby-doc.org/stdlib-2.0.0/libdoc/tempfile/rdoc/Tempfile.html not mktmpdir.  I'll make this change.